pkg/hub/client/local/client_test.go

```diff
@@ -0,0 +1,231 @@
+package local
```

Comment:
This test will be removed before merging but it's a nice showcase that implemented features work properly